galaxy/templates/configmap-nginx.yaml

```diff
@@ -61,7 +61,9 @@ data:
                 add_header X-Frame-Options SAMEORIGIN;
                 add_header X-Content-Type-Options nosniff;
             }
-
+            location /training-material/ {
+                proxy_pass https://training.galaxyproject.org/training-material/;
```

Comment:
Shouldn’t you use the value here instead of hard coding?

Reply:
I was thinking of that, but decided to hard code since the values will not change and are not something that is parametrizable.

Reply:
Not sure what I meant when I said this is not parameterizable... the value will likely never change, but it is now taken from `.Values`